templates/keepalived.conf.j2

```diff
@@ -0,0 +1,30 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
+{% for vrrp_instance in vrrp_instances %}
```

Comment:
I realize you might be pulling this from somewhere else, but once the script has for loops in it, I have to wonder if a templating language is really the most obvious way to describe the configuration.
Especially things like "if is_initial MASTER else BACKUP" wouldn't that be much clearer with a 'state' attribute in charm code instead of having it in the template?

What is pushing you to use Jinja for the logic of how to fill out the template?

IMO, it feels bad to have *some* logic in one place, decoupled from the logic in the other place.